HW_7/task_36.py

```diff
@@ -0,0 +1,15 @@
+# Задача 36: Напишите функцию print_operation_table(operation, num_rows=6, num_columns=6),
+# которая принимает в качестве аргумента функцию, вычисляющую элемент по номеру строки и
+# столбца. Аргументы num_rows и num_columns указывают число строк и столбцов таблицы,
+# которые должны быть распечатаны. Нумерация строк и столбцов идет с единицы (подумайте,
+# почему не с нуля). Примечание: бинарной операцией называется любая операция, у которой
+# ровно два аргумента, как, например, у операции умножения.
+
+def print_operation_table(operation, num_rows=6, num_columns=6):
+    x = [i + 1 for i in range(num_rows)]
+    y = [j + 1 for j in range(num_columns)]
+    
+    for i_x in x:
+       print(*(list(map(operation, [i_x] * num_columns, y))))
```

Comment:
так вывод будет чуть приятнее
```suggestion
        print(*(list(map(operation, [i_x] * num_columns, y))), sep='\t')
```